database/repository/withdraw/withdraw_test.go

```diff
@@ -153,7 +153,7 @@ func seedWithdrawData() {
 		Event(resp)
 	}
 }
-func withdrawHelper(t *testing.T) {
+func withdrawHelper(t *testing.T) { //nolint
```

Comment:
Please be more specific with your nolint usage. Here is an example from our codebase:
`//nolint:forcetypeassert // Not necessary from a pool`
we specify what we are nolinting and we provide a comment on why we are nolinting